aiokafka/producer.py

```diff
@@ -240,7 +287,8 @@ def send(self, topic, value=None, key=None, partition=None):
                 key_serializer.
 
         Returns:
-            response from Kafka
+            asyncio.Future: future object that will be set when message is
```

Comment:
So the method is a coroutine, that returns a future, is that correct?
Usage would be:
```Python
producer = AIOKafkaProducer(...)
fut = await producer.send("test", value=b"111")
result = await fut
print(result.offset)
```

Maybe it would be better to make it a function, that returns a future?